### PR TITLE
GitHub Issue Template for Releases #26

### LINKED ISSUE #26: GitHub Issue Template for Releases
Create two GitHub issue templates. A default one, and one for creating new releases. The one for releases should include all the steps associated with releasing a new gem version from CONTRIBUTING.md. The default one just needs a description header and a `closes #` placeholder for the issue it closes.

### PR DESCRIPTION
**Pull request by LittleBigProgramming**
**Description**
Resolves #26 

Added two issue templates. A default one and one for new gem releases. ✨ 

The `default` simply includes a header of description with a prompt of what to include and the `release` includes the information required to create a new gem version after a GitHub release has been made. 

Unfortunately, since this was made on a fork, which I can't raise issues on I couldn't provide any screenshots 😄  Please let me know if there are any edits you would like me to make!

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/default.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Description
+
+<!--
+Please provide a detailed description of how the issue.
```
Comment (ChaelCodes) [inline .github/ISSUE_TEMPLATE/default.md]: "how the issue" sounds like a sentence fragment. What about:
```suggestion
Please provide a detailed description of how to reproduce the issue. 
```

File: .github/ISSUE_TEMPLATE/release.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## Description
+
+
+## Steps For Releasing New Gem Version
+- Ensure all desired changes are merged to the `main` branch
```
Comment (ChaelCodes) [inline .github/ISSUE_TEMPLATE/release.md]: I dig it. Awesome. If you open this up locally to make the other change, it'd be cool to make these checkboxes, but it's good as is.

**Review by ChaelCodes — COMMENTED** [review]
If you could fix that sentence fragment, this is almost ready to merge!

File: .github/ISSUE_TEMPLATE/release.md (hunk 2 above)
Comment (LittleBigProgramming) [reply .github/ISSUE_TEMPLATE/release.md]: Updated with checkboxes instead! 🙌 

Screenshot 2022-10-23 at 14 41 22

File: .github/ISSUE_TEMPLATE/default.md (hunk 1 above)
Comment (LittleBigProgramming) [reply .github/ISSUE_TEMPLATE/default.md]: Updated 👍

**Comment by LittleBigProgramming** [comment]
Ready for re-review @ChaelCodes 😄

**Review by ChaelCodes — APPROVED** [review]
This looks great! Thank you!!

**Pull request merged**